readme.md

```diff
@@ -158,7 +158,8 @@ If you want to contribute, please read the [contribution guidelines](contributin
   - [JSON with Comments](https://github.com/ota-meshi/eslint-plugin-jsonc) - ESLint plugin for JSON, JSONC and JSON5.
   - [JSON Schema](https://github.com/ota-meshi/eslint-plugin-json-schema-validator) - Validates data defined in JavaScript, JSON, YAML and TOML using JSON Schema Validator.
 - [MDX](https://github.com/mdx-js/eslint-mdx/tree/master/packages/eslint-plugin-mdx) - ESLint Parser/Plugin for MDX.
-- [Node](https://github.com/mysticatea/eslint-plugin-node) - Additional ESLint's rules for Node.js.
+- ~~[Node](https://github.com/mysticatea/eslint-plugin-node) - Additional ESLint's rules for Node.js. No longer maintained.~~
```

Comment:
Since it's no longer maintained and the maintainer is not available, makes sense to cross out this line. WDYT?